uid-pages/layout-bonita/src/assets/css/layout.css

```diff
@@ -163,7 +163,7 @@ body > div.container-fluid > div.row:nth-child(2) iframe {
 
 .user-details--break-word p,
 .user-details--break-word pb-title:first-child {
```

Comment:
user-details--break-word should be called user-details--break-all now that we changed how it functions